Tests/Editor/Picking/RectSelectionPicker.cs

```diff
@@ -0,0 +1,138 @@
+﻿using NUnit.Framework;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.ProBuilder;
+using UnityEngine.TestTools;
+using UObject = UnityEngine.Object;
+
+/// <summary>
+/// This class ensures we generate proper lookup textures when we need them during a marquee select.
+/// These textures must not render anything than our picking objects.
+/// The setup phase will create a set of objects put at the front of a camera. As there is no selectable object,
+/// we should only get white texture as a result.
+/// Issues will likely to appear if changes have been made in graphics pipeline.
+/// This set of tests should run against standard pipeline, URP and HDRP.
+/// </summary>
+class RectSelectionPicker
+{
+    ProBuilderMesh[] selectables;
+    GameObject[] sceneObjects;
+    Camera camera;
+
+    void Setup()
+    {
+        camera = new GameObject("Camera", typeof(Camera)).GetComponent<Camera>();
+        camera.transform.position = Vector3.zero;
+        camera.transform.rotation = Quaternion.identity;
+        
+        GameObject cube = GameObject.CreatePrimitive(PrimitiveType.Cube);
+        cube.transform.position = new Vector3(0, 0, 5f);
+
+        GameObject light = new GameObject("Light", typeof(Light));
+
+        sceneObjects = new GameObject[]
+        {
+            cube,
+            light
+        };
+
+        selectables = new ProBuilderMesh[]
+        {
+        };
+
+#if UNITY_2019_1_OR_NEWER
+        foreach (SceneView sv in SceneView.sceneViews)
+            sv.drawGizmos = true;
+#endif
+    }
+
+    void Cleanup()
```

Comment:
Why is this not using [TearDown] test attribute?

Reply:
See comment above, will rectify

Reply:
Fix in 085a884